src/partials/how-its-made.html

```diff
@@ -0,0 +1,50 @@
+<main>
+  <div class="container">
+    <section>
+      <h2 class="tradition">tradition and love</h2>
+      <h1 class="how-is-made-title">how it’s made?</h1>
```

Comment:
O sa avem h1 doar in hero header, avand in vedere ca se foloseste doar 1 pe pagina, am sa te rog sa le redefinesti in h2, h3 etc.
Tot aici, te rog pentru ' sa folosesti codul pt ea. &#174; - poti sa il gasesti si pe google daca dai o cautare